README.md

```diff
@@ -16,7 +16,7 @@ Puma is a **simple, fast, multi-threaded, and highly concurrent HTTP 1.1 server
 
 Puma processes requests using a C-optimized Ragel extension (inherited from Mongrel) that provides fast, accurate HTTP 1.1 protocol parsing in a portable way. Puma then serves the request using a thread pool. Each request is served in a separate thread, so truly concurrent Ruby implementations (JRuby, Rubinius) will use all available CPU cores.
 
-Puma was designed to be the go-to server for [Rubinius](https://rubinius.com), but also works well with JRuby and MRI.
```

Comment:
Does Puma still work well with Rubinius? 🤔

IMHO we can keep the historical info (how Puma started), drop the link and state that Puma works with Ruby and JRuby.

Reply:
Or just change the link to https://github.com/rubinius/rubinius